test/JDBC/upgrade/latest/schedule

```diff
@@ -538,3 +538,4 @@ babel_4328_datetimeoffset
 babel_726
 BABEL-3401
 PARTITION
+BABEL-5071
```

Comment:
Could you add this to other schedule files? For example, the prepare script is executed in a previous version and the verify script is executed in the latest version.

Reply:
done